evcouplings/visualize/pairs.py

```diff
@@ -1183,43 +1182,101 @@ def enrichment_pymol_script(enrichment_table, output_file,
         "cartoon putty"
     chain : str, optional (default: None)
         Use this PDB chain in residue selection
+    legacy: bool, optional (default: False)
+        Use legacy (2011) red and yellow colormap
+        for EC enrichment
     """
-    t = enrichment_table.query("enrichment > 1")
+    if legacy: 
+        t = enrichment_table.query("enrichment > 1")
+        t.loc[:, "b_factor"] = t.enrichment
+        # compute boundaries for highly coupled residues
+        # that will be specially highlighted
+        boundary1 = int(0.05 * len(t))  # top 5%
+        boundary2 = int(0.15 * len(t))  # top 15%
+
+        t.loc[:, "b_factor"] = t.enrichment
+
+        # set color for "low" enrichment (anything > 1)
+        t.loc[:, "color"] = "yelloworange"
 
-    # compute boundaries for highly coupled residues
-    # that will be specially highlighted
-    boundary1 = int(0.05 * len(t))  # top 5%
-    boundary2 = int(0.15 * len(t))  # top 15%
+        # high
+        t.loc[t.iloc[0:boundary1].index, "color"] = "red"
 
-    t.loc[:, "b_factor"] = t.enrichment
+        # medium
+        t.loc[t.iloc[boundary1:boundary2].index, "color"] = "orange"
+        
+    else:
+        t = deepcopy(enrichment_table)
+        t.loc[:, "b_factor"] = t.enrichment
+
+        # set boundaries for enrichment levels
+        # that will be specially highlighted
+        # create nine subsets
+        boundary_list = [
+                int(0.11 * len(t)),
+                int(0.22 * len(t)),
+                int(0.33 * len(t)),
+                int(0.44 * len(t)),
+                int(0.55 * len(t)),
+                int(0.66 * len(t)),
+                int(0.77 * len(t)),
+                int(0.88 * len(t)),
+                int(1.00 * len(t))
+            ]
+
+        # list of colors to color each category
+        # must be same length and order as boundary_list
+        # list of rgb tuples
+        color_list = [
+            (77, 0, 75),  # dark purple
+            (129, 15, 124),
+            (136, 65, 157),
+            (140, 107, 177),
+            (140, 150, 198),
+            (158, 188, 218),
+            (191, 211, 230),
+            (224, 236, 244),
+            (247, 252, 253)  # almost white
+        ]
 
-    # set color for "low" enrichment (anything > 1)
-    t.loc[:, "color"] = "yelloworange"
+        # convert to fractions
+        color_list = [(x/255., y/255., z/255.) for x, y, z in color_list]
```

Comment:
Minor details: `[(x / 255, y / 255, z / 255) for x, y, z in color_list]`

In Python 3 floating point division is the default so it should work without the .

Reply:
Oh that's a good default behavior, I will change because it looks cleaner